packages/web/src/components/layouts/HomeNew/HomeNew.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import { CardProps } from '../../ui/Card/Card';
+import Hero from '../../ui/Hero/Hero';
+import Container from 'components/ui/Container/Container';
+import CardList from '../../ui/CardList/CardList';
+import CardGroups from '../../ui/CardGroups/CardGroups';
+import Footer from '../../ui/Footer/Footer';
+import CardGroup from '../../ui/CardGroup/CardGroup';
```

Comment:
A few thoughts about component structure:
- I'd suggest having a simple `index.ts` reexporting the component in each component folder so we could just `import { Component } from '../../ui/Component'`.
- I would also recommend against `export default`ing components in favor of named exports.
- I'd ask for better uniformity of import paths, e.g. import either from `components/ui/...` or from `../../ui/...`, but not both.

PS. Not sure if this is really required to be fixed in this PR, we can do it later actually.